### ISSUE TITLE
Remove duplicate content files in MusicStore.Standalone

### ISSUE THREAD

**Issue by JunTaoLuo** [aspnet/MusicStore#650]
Once https://github.com/dotnet/cli/pull/2493 is available we can remove the duplicate content files I added to MusicStore.Standalone in https://github.com/aspnet/MusicStore/pull/645. The update in the cli will allow us to map out-of-directory content files to the publish output directory correctly.

**Comment by JunTaoLuo** [comment]
Blocked on https://github.com/dotnet/cli/issues/2861

**Comment by JunTaoLuo** [comment]
Won't fix for rc2 since dotnet/cli issue is moved to rtm.
